### ISSUE TITLE
Octodash image as splash 

### ISSUE THREAD

**Issue by dmiart** [UnchartedBull/OctoDash#909]
**Is your feature request related to a problem? Please describe.**
Replace  /usr/share/plymouth/themes/pix/splash.png with Octodash themed image.

**Describe the solution you'd like**
Replace  /usr/share/plymouth/themes/pix/splash.png with Octodash themed image as an option in the installation wizard. 

![splash](https://user-images.githubusercontent.com/33436796/90323491-fe3ba480-df2f-11ea-8abb-d4b1d7186bcc.png)
![splash](https://user-images.githubusercontent.com/33436796/90323500-10b5de00-df30-11ea-90f8-92be1e1460be.png)

Something like this. 

![image](https://user-images.githubusercontent.com/33436796/90323520-87eb7200-df30-11ea-9a83-1bf02b646e5b.png)

**Comment by UnchartedBull** [comment]
That would then replace the pi logo during boot, right? Looks cool! Linked to #897 kind of to make a fully OctoDash themed distro.

**Comment by dmiart** [comment]
> 
> 
> That would then replace the pi logo during boot, right? Looks cool! Linked to #897 kind of to make a fully OctoDash themed distro.

Yeah, this replaces the welcome to raspbian desktop picture. Also have to enable the splash display in raspi-config.

**Comment by TheSin-** [comment]
Don't forget to make sure rpd-plym-splash is installed

**Comment by TheSin-** [comment]
Also add

disable_splash=1

to config.txt

and add

logo.nologo vt.global_cursor_default=0

to cmdline.txt

this will make it truly silent

https://scribles.net/silent-boot-up-on-raspbian-stretch/

**Comment by dmiart** [comment]
What if you send cls in couple of spots before ratpoison? I can't test on mine as I am printing atm. I suspect the text will still be briefly visible though.

**Comment by TheSin-** [comment]
Okay I finally did it based on this link

https://scribles.net/silent-boot-on-raspbian-stretch-in-console-mode/

touch ~/.hushlogin

and

in /etc/systemd/system/getty@tty1.service.d/autologin.conf replace

ExecStart=-/sbin/agetty --autologin pi --noclear %I xterm-256color

with

ExecStart=-/sbin/agetty --skip-login --noclear --noissue --login-options "-f pi" %I $TERM

and my one addition is in .bashrc change

source /home/pi/scripts/welcome
if [ -z "$SSH_CLIENT" ] || [ -z "$SSH_TTY" ]; then
    xinit -- -nocursor
fi

to

if [ -z "$SSH_CLIENT" ] || [ -z "$SSH_TTY" ]; then
    xinit -- -nocursor > /dev/null 2>&1
else
    source /home/pi/scripts/welcome
fi

This produces a few second blank screen while ratpoison loads but zero text at all.

I also had to make the splash screen 1280x800 for my display and I made it all grey without the black but that is just preference, this looks amazing, if you want I can make a video of the boot for anyone that is curious.

great work and idea on this @dmiart

**Comment by TheSin-** [comment]
https://youtu.be/uOcAkeExMWU

**Comment by dmiart** [comment]
Yeah, this is why I had the grey rectangle that was basically a copy paste of the original but with different colors to fit all resolutions. But it does look much better with all grey tbh.

**Comment by UnchartedBull** [comment]
Holy sh*t that looks awesome! Kind of looks like a fully integrated system now, rather than something hacky. Really appreciate your efforts! I'll try to integrate that into the installation script and merge it over in the wiki so other people can also adjust there systems :)

**Comment by TheSin-** [comment]
yeah it looks really good, it was a great idea @dmiart had.
